front-end/src/screens/CreatePost/index.tsx

```diff
@@ -85,7 +113,7 @@ const CreatePost = ({ className }:Props): JSX.Element => {
 
 	const onTitleChange = (event: React.ChangeEvent<HTMLInputElement>[]) => {setTitle(event[0].currentTarget.value); return event[0].currentTarget.value;};
 	const onContentChange = (data: Array<string>) => {setContent(data[0]); return data[0].length ? data[0] : null;};
-	// const onPollChanged = (event: React.FormEvent<HTMLInputElement>, data: CheckboxProps) => { setHasPoll(data.checked || false);};
+	const onPollChanged = (event: React.FormEvent<HTMLInputElement>, data: CheckboxProps) => { setHasPoll(data.checked || false);};
```

Comment:
Isn't `data.checked` a boolean? the `|| false` should probably not be needed here.

Reply:
data.checked is boolean | undefined